src/main/resources/static/config/defaultProperties.xml

```diff
@@ -35,6 +37,11 @@
             <pair>AVG;MITTELWERT</pair>
         </aggregatefunctionlang>
     </synonyms>
+    <yesorno>
```

Comment:
nahhhh we dont call it "yesorno", lets call it specificoptions or something like this

Reply:
changed